vote_text_drafts/rpip-28.md

```diff
@@ -0,0 +1,29 @@
+> Vote options:
+> - For
+> - Against
+> - Abstain
+
+# Deposits Under the Minimum (RPIP-28)
+
+### Outcomes
+ If this vote passes, it will be possible to make a minipool deposit when under the "minimum" RPL threshold by depositing the ETH and staking (at least) the minimum amount of RPL needed for a single minipool of that type (this is in _addition_ to the current method). Note that this requires a smart contract change, so it won't take effect until it is included in a larger smart contract release.
```

Comment:
```suggestion
If this vote passes, it will indicate the pDAO's support of the functionality described in RPIP-28. Once implemented, this will make it possible to make a minipool deposit when under the "minimum" RPL threshold by depositing the ETH and staking (at least) the minimum amount of RPL needed for a single minipool of that type. This is in _addition_ to the current method.
 
 This will require a smart contract change, so it will not take effect until it is included in a larger smart contract release.
```

I struggled to reword this usefully, maybe this?

Reply:
Took a different stab at it -- have a look when pushed